<?xml version="1.0" encoding="UTF-8" standalone="no"?><testsuite errors="0" failures="2" name="Importar pausas" skipped="0" tests="7" time="97.15" timestamp="2023-03-07 03:39:47"><testcase name="Importar un archivo con formato invalido"/><testcase name="Importar un archivo con columnas faltantes"/><testcase name="Importar un archivo con columnas repetidas"/><testcase name="Importar un archivo con columnas extras"/><testcase name="Importar un archivo con columnas sin nombre"/><testcase name="Importar un archivo con datos invalidos"><failure message="&#10;Expecting a value that contains: &lt;[already_exists_dtmf]&gt;&#10;But got........................: &lt;&quot;&quot;&gt;" type="java.lang.AssertionError">
Expecting a value that contains: &lt;[already_exists_dtmf]&gt;
But got........................: &lt;""&gt;</failure><system-err>
Expecting a value that contains: &lt;[already_exists_dtmf]&gt;
But got........................: &lt;""&gt;&#13;
co.com.arbusta.certification.expand.interactions.errorListTable.verificarErroresTabla(errorListTable.java:18)&#13;
co.com.arbusta.certification.expand.stepdefinitions.administration.basic.pauses.importPauses_Steps.se_permite_descargar_el_archivo_de_errores(importPauses_Steps.java:97)&#13;
io.cucumber.java.Invoker.doInvoke(Invoker.java:66)&#13;
io.cucumber.java.Invoker.invoke(Invoker.java:24)&#13;
io.cucumber.java.AbstractGlueDefinition.invokeMethod(AbstractGlueDefinition.java:47)&#13;
io.cucumber.java.JavaStepDefinition.execute(JavaStepDefinition.java:29)&#13;
io.cucumber.core.runner.CoreStepDefinition.execute(CoreStepDefinition.java:66)&#13;
io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:63)&#13;
io.cucumber.core.runner.TestStep.executeStep(TestStep.java:86)&#13;
io.cucumber.core.runner.TestStep.run(TestStep.java:57)&#13;
io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:51)&#13;
io.cucumber.core.runner.TestCase.run(TestCase.java:95)&#13;
io.cucumber.core.runner.Runner.runPickle(Runner.java:75)&#13;
io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)&#13;
io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)&#13;
io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)&#13;
io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:144)&#13;
io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:28)&#13;
io.cucumber.junit.FeatureRunner.run(FeatureRunner.java:137)&#13;
io.cucumber.junit.CucumberSerenityRunner.runChild(CucumberSerenityRunner.java:258)&#13;
io.cucumber.junit.CucumberSerenityRunner.runChild(CucumberSerenityRunner.java:58)&#13;
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)&#13;
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
</system-err></testcase><testcase name="Importar un archivo con datos validos"><failure message="&#13;&#10;Expected: an element that is clickable&#13;&#10;     but: no matching element found by [The button for use all filters located with: //expand-shared-filter-button-field[1]/expand-shared-button-field[1]/div[1]/button[1]] was not clicked" type="java.lang.AssertionError">&#13;
Expected: an element that is clickable&#13;
     but: no matching element found by [The button for use all filters located with: //expand-shared-filter-button-field[1]/expand-shared-button-field[1]/div[1]/button[1]] was not clicked</failure><system-err>&#13;
Expected: an element that is clickable&#13;
     but: no matching element found by [The button for use all filters located with: //expand-shared-filter-button-field[1]/expand-shared-button-field[1]/div[1]/button[1]] was not clicked&#13;
co.com.arbusta.certification.expand.interactions.searchInTheBar.performAs(searchInTheBar.java:44)&#13;
co.com.arbusta.certification.expand.stepdefinitions.administration.basic.pauses.importPauses_Steps.se_realiza_correctamente_la_importacion_de_los_archivos(importPauses_Steps.java:108)&#13;
io.cucumber.java.Invoker.doInvoke(Invoker.java:66)&#13;
io.cucumber.java.Invoker.invoke(Invoker.java:24)&#13;
io.cucumber.java.AbstractGlueDefinition.invokeMethod(AbstractGlueDefinition.java:47)&#13;
io.cucumber.java.JavaStepDefinition.execute(JavaStepDefinition.java:29)&#13;
io.cucumber.core.runner.CoreStepDefinition.execute(CoreStepDefinition.java:66)&#13;
io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:63)&#13;
io.cucumber.core.runner.TestStep.executeStep(TestStep.java:86)&#13;
io.cucumber.core.runner.TestStep.run(TestStep.java:57)&#13;
io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:51)&#13;
io.cucumber.core.runner.TestCase.run(TestCase.java:95)&#13;
io.cucumber.core.runner.Runner.runPickle(Runner.java:75)&#13;
io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)&#13;
io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)&#13;
io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)&#13;
io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:144)&#13;
io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:28)&#13;
io.cucumber.junit.FeatureRunner.run(FeatureRunner.java:137)&#13;
io.cucumber.junit.CucumberSerenityRunner.runChild(CucumberSerenityRunner.java:258)&#13;
io.cucumber.junit.CucumberSerenityRunner.runChild(CucumberSerenityRunner.java:58)&#13;
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)&#13;
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
</system-err></testcase></testsuite>